packages/ibm-products/src/components/ExportModal/ExportModal.tsx

```diff
@@ -204,6 +205,15 @@ export let ExportModal = forwardRef(
       }
     }, [filename, preformattedExtensions, open]);
 
+    useEffect(() => {
+      if (successful) {
+        const button: HTMLButtonElement | null = document.querySelector(
+          `.${carbon.prefix}--modal-close-button button`
```

Comment:
Hey @sangeethababu9223, can you use block class here, avoid using classes prefixed with carbon.prefix directly.

Reply:
Thanks @anamikaanu96!
This is updated now.